index.html

```diff
@@ -0,0 +1,319 @@
+          <a class="nav-link navlink" href="#">Logout</a>
+        </li>
+      </ul>
+    </nav>
+
+    <!-- // navbar Section // -->
+
+    <nav class="navbar navbar-expand-sm bg-light navbar-light">
+      <div class="container">
+        <a class="navbar-brand" href="./index.html">
+          <img src="./images/Enha (1).png" alt="logo" id="logoMain" />
+        </a>
+        <button
+          class="navbar-toggler main_btn"
+          type="button"
+          data-bs-toggle="collapse"
+          data-bs-target="#collapsibleNavbar"
+        >
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div
+          class="collapse navbar-collapse justify-content-end"
+          id="collapsibleNavbar"
+        >
+          <ul id="menyOptions" class="navbar-nav nav2">
+            <li class="nav-item">
+              <a class="nav-link" href="./index.html">Home</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="./pages/about.html">About</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#spearkers">Speakers</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#partners">Partners</a>
+            </li>
+          </ul>
+          <ul id="popup" class="navbar-nav nav2">
+            <div id="cross-icon">
+              <i
+                onclick="off()"
+                class="material-icons"
+                style="font-size: 50px; color: rgb(226, 215, 215);"
+              >
+                close
+              </i>
+            </div>
+            <li onclick="off()" class="nav-item">
+              <a class="nav-link li-nav" href="./index.html">Home</a>
+            </li>
+            <li onclick="off()" class="nav-item">
+              <a class="nav-link li-nav" href="./pages/about.html">About</a>
+            </li>
+            <li onclick="off()" class="nav-item">
+              <a class="nav-link li-nav" href="#spearkers">Speakers</a>
+            </li>
+            <li onclick="off()" class="nav-item">
+              <a class="nav-link li-nav" href="#partners">Partners</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+
+    <!-- // Main Section // -->
+
+    <div id="main1" class="container-fluid">
+      <div class="container py-5 px-1">
+        <h2 class="mainText">"Hello, Sharing World"</h2>
+        <h1 class="mainText">
+          Enhance Your Skills
+          <br />
+          GLOBAL SUMMIT
+          <br />
+          2022
+        </h1>
+      </div>
+
+      <div class="container mainPara text-black my-3">
+        <p>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Fugit ad
+          laudantium voluptatum, modi natus perspiciatis. Lorem ipsum dolor sit
+          amet consectetur adipisicing elit.
+        </p>
+      </div>
+      <div class="container mt-3">
+        <h1>
```

Comment:
- [ OPTIONAL ] Great job but I think here in your main page there is a big gap between the heading and the text so please decrease it to make your design match the original design ✅🔥

- [ OPTIONAL ] I think it would be great if you make you heading here has more fornt weight since it is not the same for weight as the original design 🚀💯💯♻

## Notice here the second heading has  ore font wight 🙏🏻💯💯
- Your design
![image](https://user-images.githubusercontent.com/58553711/205466505-418ba320-0811-4373-8902-4d24a9edb669.png)

- Given design
![image](https://user-images.githubusercontent.com/58553711/205466525-c5032c1c-02c6-407a-a66a-cc4651b36ffc.png)